getting_started/scripting/visual_script/index.rst

```diff
@@ -8,4 +8,4 @@ VisualScript
    what_is_visual_scripting
    getting_started
    nodes_purposes
-
+   custom_visualscript_nodes
```

Comment:
Remove the blank line above `custom_visualscript_nodes`.

Reply:
I think this is fixed now.